crates/utils/src/config.rs

```diff
@@ -17,18 +17,29 @@ pub const DEFAULT_BLOCK_PRODUCER_PORT: u16 = 48046;
 pub const DEFAULT_STORE_PORT: u16 = 28943;
 pub const DEFAULT_FAUCET_SERVER_PORT: u16 = 8080;
 
+#[derive(Clone, Eq, PartialEq, Ord, PartialOrd, Hash, Debug, Serialize, Deserialize)]
+pub enum Protocol {
+    Http,
```

Comment:
I think impl `Default` here. And then replace the `protocol: Protocol::Http` with `Default::default()`. This way we can change `Http` to `Https` default by only changing the enum's default.

```suggestion
    #[default]
    Http,
```